content/04.results.md

```diff
@@ -151,7 +151,37 @@ The horizontal lines represent the overall proportion of preprints  that are pub
 We overlaid these new annotations onto existing annotations to reassess the overall preprint publication rate reported by Abdill et al. [@doi:10.7554/eLife.45133].
 Our filtering criteria were intentionally stringent, so the increased estimate of publication rate amounts to a few percent (Figure {@fig:updated_pub_rate}).
 Many of these missed annotations were for preprints posted in the 2017-2018 interval.
-Compared to preprints published in 2019 and later, the preprints posted in 2017-2018 are old enough to have a high chance of being published; however, it is interesting that the rate for older preprints was not observed to be higher.
+Compared to preprints published in 2019 and later, the preprints posted in 2017-2018 are old enough to have a high chance of being published; however, it is interesting that the rate for older preprints was not observed to be higher.
+
+### Measuring Time Delay for Preprint Publication Process
+
+![
+On average preprints suffer a 17-day delay before they are published.
+This squarebin plot depicts the amount of time it takes a preprint to be published against the distances of a preprint's first version and its corresponding published version.
+The x-axis represents the Euclidean distance between document representations, while the y-axis represents the amount of days elapsed between a preprint posted on bioRxiv and the time a preprint is published.
+The color bar on the right represents the density of each square-bin in this plot where more dense regions have a brighter color compared to their counterparts.
+](https://raw.githubusercontent.com/danich1/annorxiver/c5b782b91e388dd1143f0b3e42c59a1c8c3277e1/biorxiv/publication_delay_experiment/output/article_distance_vs_publication_time.png){#fig:distance_publication_time}
+
+![
+Preprints with multiple versions suffer an average of a 51-day delay to get published.

```

Comment:
Is this that each version takes an additional 51 days? I think that's the thing you're looking at here. Not that just posting 2 or more versions adds 51 days